packages/react-core/src/helpers/OUIA/OUIA.md

```diff
@@ -0,0 +1,94 @@
+---
+id: Open UI Automation
+section: developer-resources
+---
+
+## About Open UI Automation
+
+Open UI Automation (OUIA) is a specification of certain key guidelines to follow when creating a new
+web framework or application. The goal is to ease the burden of individuals wishing to create and
+maintain automated testing environments. OUIA standardizes some HTML attributes and web page
+behavior. It helps to create trivial and unambiguous locators of the elements in the DOM for
+various automated test suites.
+
+### Why not use `id` or `ARIA`?
+
+`ARIA` is designed as an accessibility specification and it doesn't account aspects that are crucial
+for automation such as a component type. `id` attribute might be reserved by some front-end code and
+there might be restrictions of changing it. Having a separate id attribute makes it application and
+framework independent. Developers and testers have a contract that these attributes are designed for
+UI automation.
+
+### Specification parts
+
+The following blocks exist in the specification:
+
+* `OUIA:Component` - components and component frameworks
+* `OUIA:Page` - page identification
+* `OUIA:PageSafe` - interaction safety
+
+You can find the complete description on the [OUIA website](https://ouia.readthedocs.io).
+
+[Selected](#list-of-ouia-compliant-components) PatternFly 4 components are `OUIA:Component`
+compliant.
+
+### Usage
+
+PatternFly 4 generates a `data-ouia-component-type` attribute for selected components automatically
+based on the name. For example, the `data-ouia-component-type` attribute for the
+[button component](/components/button) is `PF4/Button`. The id of `data-ouia-component-id`
+attribute is random by default. To specify the concrete id, you should add an `ouiaId` prop to the
+component.
+
+### Example
+
+```js
+import React from 'react';
+import { Button } from '@patternfly/react-core';
+
+<React.Fragment>
+  <Button>Button</Button>
+  <br />
+  <br />
+  <Button ouiaId="static_id">Button with ouiaId</Button>
+</React.Fragment>
+```
+
+## List of OUIA compliant components
```

Comment:
See above comment on deleting the line about the list and changing this heading to "OUIA-compliant PatternFly 4 components"

Reply:
Done.